### ISSUE TITLE
1.3.2 breaks console

### ISSUE THREAD

**Issue by boboldehampsink** [pierrestoffe/craft-language-redirector#15]
Since 1.3.2 I get the following error when running console commands, as it initializes the request service while that doesn't exist in console

```
Exception 'yii\base\UnknownMethodException' with message 'Calling unknown method: craft\console\Request::getQueryStringWithoutPath()'

```

**Comment by pierrestoffe** [comment]
@boboldehampsink I just tested on my local environment and I can't replicate the issue. Also I don't use console commands often so I might be missing something obvious. Could you give me an example of a console command that triggers the error?

Edit: ok, I managed to get it when running migrations

**Comment by pierrestoffe** [comment]
Fixed in [v1.3.3](https://github.com/pierrestoffe/craft-language-redirector/releases/tag/1.3.3) that was just released, sorry about that.

**Comment by boboldehampsink** [comment]
Fixed, thanks!
